Arduino/SensorLogic/SensorLogic.ino

```diff
@@ -148,7 +159,37 @@ void Fingerprintloop()
   //begin to check if the finger is pressed, and wake up the module and match it
   if(Finger_SleepFlag == 1)
   {
-    Auto_Verify_Finger();
+    if(digitalRead(Finger_WAKE_Pin) == HIGH)   // If you press your finger
+  { 
+    delay(20);      
```

Comment:
Auto_Verify_Finger() is replaced with the code extracted from the library for flexibility. This allows the use the Servoloop(int dir) function to turn the lock upon a successful match in Sleep Mode.